panels/site_sections/schedule.py

```diff
@@ -0,0 +1,194 @@
+from panels import *
```

Comment:
This file was moved here from the core repo and is otherwise unchanged (save for the single ``import`` line above).